### ISSUE TITLE
How to Save Model and Save CheckPoint in tf.keras mode

### ISSUE THREAD

**Issue by Johnson-yue** [kashif/tf-keras-tutorial#3]
Hi, your tutorial is very helpful to me!!! Thank you very much.
I read your code many times， and I found it is not include __How to Save Model__, as I know, since tf 1.10 and tf-serving out,  Tensorflow-Team using SavedModel more than Checkpoint for Saving model.

Second, Can you write some tutorial about using tensorboad with tf.keras ?
I can use tensorboard with session , and I can save gradient for debug. But  in tf.keras  what should I do?

**Comment by kashif** [comment]
thanks @Johnson-yue i will try to find time to add the things you requested...
